bitrise.yml

```diff
@@ -36,7 +36,13 @@ workflows:
             fi
   test:
     steps:
-    - go-list:
-    - golint:
-    - errcheck:
-    - go-test:
+    - script@1:
```

Comment:
NOTE: this is shared across many repos, last time we had to update every occurrence, due to a specific version being removed. I think it would be worth creating a step for this script sooner or later.